app/importers/file_importers/downloader.rb

```diff
@@ -0,0 +1,32 @@
+class Downloader
+
+  def initialize(log:, client:, remote_file_name:, transformer:)
+    @log = log
+    @client = client
+    @remote_file_name = remote_file_name
+    @transformer = transformer
+  end
+
+  def get_data
+    file = download_file
+
+    data = @transformer.transform(file)
+
+    CleanupReport.new(
+      @log, @remote_file_name, @transformer.pre_cleanup_csv_size, data.size
+    ).print
+
+    return data
+  end
+
+  def download_file
```

Comment:
Is this intended to be public or private?

Reply:
@kevinrobinson — Private, I'm thinking `get_data` should be the only public method. Will add that.